api/src/main/java/io/opentelemetry/trace/SpanContext.java

```diff
@@ -136,13 +136,13 @@ public String getSpanIdAsHexString() {
     return SpanId.bytesFromHex(getSpanIdHex(), 0);
   }
 
-  /**
-   * Returns the {@code TraceFlags} associated with this {@code SpanContext}.
-   *
-   * @return the {@code TraceFlags} associated with this {@code SpanContext}.
-   * @since 0.1.0
-   */
-  public abstract TraceFlags getTraceFlags();
+  /** Whether the span in this context is sampled. */
+  public abstract boolean isSampled();
+
+  public void copyTraceFlagsHexTo(char[] dest, int destOffset) {
```

Comment:
Do you think we can just keep this in the w3c propagator? We previously had a data object that corresponded with the w3c concept of traceflags, but now if we don't, it seems we don't need this here in the first place. I guess this comment applies to the entire `TraceFlags` class itself too.

Reply:
I think that seems fair to me, yes.

Reply:
Let's consider that as a follow-on change, if you're ok with that. TraceFlags is used in other places that just the w3c propagator right now.